common/djangoapps/external_auth/views.py

```diff
@@ -546,7 +546,7 @@ def shib_login(request):
     )
 
 
-def _safe_postlogin_redirect(redirect_to, safehost, default_redirect='/'):
+def _safe_postlogin_redirect(redirect_to, localhost, default_redirect='/'):
     """
     If redirect_to param is safe (not off this host), then perform the redirect.
```

Comment:
can you update the docstring and within the updated docstring explain the logic?